Logger/LogFactory.cs

```diff
@@ -1,10 +1,28 @@
-﻿namespace Logger;
+﻿using System;
+#nullable enable
+using System.IO;
+namespace Logger;
 
 public class LogFactory
 {
-    public BaseLogger CreateLogger(string className)
+    private string? _FilePath;
+    public BaseLogger? CreateLogger(string className)
     {
+        if (_FilePath == null)
+        {
 
-        return null;
+            return null;
+        }
+        else {
+
+          BaseLogger logger = new FileLogger(_FilePath);
+            return logger;
```

Comment:
This can be done in a single line
```suggestion
          return new FileLogger(_FilePath);
```